static/panes/compiler.js

```diff
@@ -914,6 +925,18 @@ Compiler.prototype.onGccDumpFiltersChanged = function (id, filters, reqCompile)
     if (this.id === id) {
         this.treeDumpEnabled = (filters.treeDump !== false);
         this.rtlDumpEnabled = (filters.rtlDump !== false);
+        this.ipaDumpEnabled = (filters.ipaDump !== false);
+
+        this.addressOptionEnabled = (filters.addressOption !== false);
```

Comment:
If you want to avoid copying big chunks of bools we could build a single object here.

```javascript
this.dumpFlags = {
    slim: filters.slimOption !=== false,
    raw: filers.rawOption !=== false,
    ...etc
};
```

and then you just have to deal with a single `dumpFlags` object everywhere else.